cmd/runj-entrypoint/main.go

```diff
@@ -121,6 +122,14 @@ func dupStdio(slavePath string) error {
 			Err:  err,
 		}
 	}
+
+	if _, err := syscall.Setsid(); err != nil {
+		return err
+	}
+
+	if ret, _, err := unix.Syscall(unix.SYS_IOCTL, uintptr(fd), unix.TIOCSCTTY, uintptr(0)); ret != 0 || err != 0 {
+		return fmt.Errorf("Unable to set controlling tty")
```

Comment:
Wouldn't it make sense to do something with `ret` and/or `err` here depending on which condition got us to this point?

(For example, isn't `err` going to be an `error` object?  Meaning it could be wrapped with something like `fmt.Errorf("Unable to set controlling tty: %w", err)`?)

Reply:
Yes I should ;) err is an errno, and that could be wrapped as well. 
I think ret is actually safe to ignore, because I think errno would always be set when ret would be non zero

Reply:
I went with returning err instead of formatting, as that is what the rest of the block does as well.